recipes/install_server.rb

```diff
@@ -38,11 +38,18 @@
     action [:enable, :start]
 end
 
+file "/etc/cron.d/novnc_tokens" do
+    owner "root"
+    group "root"
+    mode  "0644"
+    action :create
+end
```

Comment:
Just one last thing left! Add a unit test to verify that this file is created in the `install_server_spec.rb`.